packages/smart-wallet/test/test-invitation1.js

```diff
@@ -33,6 +33,7 @@ const mockBootstrapPowers = async (
   log,
   spaceNames = ['installation', 'instance', 'issuer', 'brand'],
 ) => {
+  /** @type {import('@agoric/vat-data').Baggage} */
   const baggage = makeScalarMapStore('bootstrap');
```

Comment:
Does anybody know why the type of `baggage` isn't inferred from `makeScalarMapStore`?

Reply:
`Baggage` is `MapStore<string, any>` but `makeScaleMapStore` returns `MapStore<unknown, unknown>`.

It can take `keyShape` and `valueShape` in options, so it could eventually infer those two type parameters, but that would depend on the work to infer types from shapes.

Reply:
> that would depend on the work to infer types from shapes

adding a link to track motivations:

 - https://github.com/Agoric/agoric-sdk/issues/6160